src/events/2018-03-14.md

```diff
@@ -33,11 +33,13 @@ agenda:
     description: Chat, mingle and soak in all you've learned so far
   - time: 8:30pm
     type: speaker
-    title: "[Ruth John](https://twitter.com/rumyra) - Title TBC"
+    title: "[Ruth John](https://twitter.com/rumyra) - Web Workers"
     description: |
       Ruth will talk about web workers.
 
-      Full talk description and bio TBC.
```

Comment:
Stripped out the TBC stuff and popped something there 'cos the event is in just a couple of days so let's get something there.